### ISSUE TITLE
[UX] Move file type settings to Configuration > Media

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#4583]
I just discovered the File type settings page at `admin/structure/file-types/settings` and thought it looked similar to the settings on `admin/config/media/file-system`.

Would it make more sense to people to put all their settings for uploaded files in the same place? I understand that one is settings for files in general, and the other is for file_entity settings, but I'm not sure the average person would know the difference (or care?)

feedback welcome!

**Comment by klonos** [comment]
The former applies to file types, while the latter to the file system settings. I wouldn't object consolidating those though. So 👍 from me.

**Comment by olafgrabienski** [comment]
> Would it make more sense to people to put all their settings for uploaded files in the same place?

Not sure, but there is one more place which might make it more difficult: In text format configuration (e.g. `admin/config/content/formats/filtered_html`) you can "Enable file uploads", set an Upload directory and more. It's quite hidden, and I often forget this option when building new sites.

**Comment by klonos** [comment]
> ...there is one more place which might make it more difficult: In text format configuration...

That should be a separate UX issue @olafgrabienski. The settings you are referring to are per-format, so that each text format may have its own settings for images/files. That means that they can't be all in the same place.

**Comment by olafgrabienski** [comment]
> The settings you are referring to are per-format, so that each text format may have its own settings ...

Yep, that's why I see the goal to "put **all** their settings for uploaded files in the same place" as difficult. Maybe it's even not possible if we consider text format settings. I mentioned them however here to make us aware of them in the given context.

**Comment by klonos** [comment]
@olafgrabienski #4585 😉

**Comment by jenlampton** [comment]
I really dislike that the destination of files uploaded via the editor is configured on the text format pages. I'd prefer that those be moved somewhere else (and a link added in the text format section).  I still think that Configuration > Media is the right place for those. 

Then, if any other module (like IMCE) wants to define where other files can go, they could form-alter that form to add their own settings.

I would also like to see more options for where files uploaded via the editor can go, like the ability to change the location based on content type. 

I'm not sure yet how to make that happen, but I think it would be possible for a contrib module to do it.
